src/visions/backends/spark/types/boolean.py

```diff
@@ -0,0 +1,11 @@
+from pyspark.sql.dataframe import DataFrame
+from pyspark.sql.types import BooleanType
+
+from visions.types.boolean import Boolean
+
+
+@Boolean.contains_op.register
+def boolean_contains(sequence: DataFrame, state: dict) -> bool:
+    assert len(sequence.schema) == 1
```

Comment:
Caveat: not a spark expert (so apologies if this is a dumb question).

Why do we produce a runtime error if the sequence has more than one associated schema rather than returning `False`?